conf/Makefile.chibios

```diff
@@ -95,6 +105,16 @@ ifeq ($(USE_LINK_GC),)
   USE_LINK_GC = yes
 endif
 
+# Linker extra options here.
+ifeq ($(USE_LDOPT),)
+  USE_LDOPT = 
```

Comment:
Probably better to make it consistent with the original ChibiOS examples, at least for now